### ISSUE TITLE
fr(MatSlider): Support custom ARIA values for now/min/max.

### ISSUE THREAD

**Issue by pope** [angular/components#19087]
#### Feature Description

For the MatSlider component, users should be able to set how ARIA min/max/now values are communicated to the screen reader. Right now, it just reflects the `min`, `max`, and `value` of the slider; but we would like to contextualize the values.

This feature could be implemented similarly to how `displayValue` is done. I'm not sure we would want to re-use `displayValue` for  `aria-valuenow` though since it could cause a regression for existing users. But I don't have enough context to really say otherwise.

#### Use Case

This would allow developers to add contextual information to the slider, similar to how `displayValue` is used.

Say `MatSlider` is used for an audio scrub bar, where the min is 0, the max is the length of some audio track, and the value now is the playhead. As the user moves the scrub bar this feature could enable human-friendly labels. So instead of using the slider having a screen reader value of "80.0", we could have a value like "1 minute, 20 seconds".

**Comment by pope** [comment]
BTW, I am down for adding the code to this feature, I just want to make sure this is agreeable first.

**Comment by crisbeto** [comment]
This sounds reasonable, but [looking at the docs](https://www.w3.org/TR/wai-aria-1.1/#aria-valuemin), `aria-valuemin` and `aria-valuemax` are only allowed to be numbers.

**Comment by pope** [comment]
Yes. You are very right. It seems I missed `aria-valuetext`. Closing.
